addons/mass_mailing/models/mailing.py

```diff
@@ -650,3 +654,53 @@ def _image_to_url(b64image: bytes):
             return lxml.html.tostring(root)
 
         return body_html
+
+    def _send_statistics_mails(self):
+        """Send an email to the responsible of each finished mailing with the statistics."""
+        if not self.env['ir.config_parameter'].sudo().get_param('mass_mailing.send_kpi_email'):
+            return
+
+        mailings = self.env['mailing.mailing'].search([
+            ('kpi_mail_sent', '=', False),
+            ('state', '=', 'done'),
+            ('sent_date', '<=', fields.Datetime.now() - relativedelta(minutes=5)),
+            ('sent_date', '>=', fields.Datetime.now() - relativedelta(days=3)),
+        ])
+
+        if not mailings:
+            return
+
+        mail_template = self.env.ref('mass_mailing.mail_template_mass_mailing_kpi')
+        if not mail_template:
+            _logger.error('Mass Mailing: can not found the mass mailing KPI template')
+            return
+
+        mailings.kpi_mail_sent = True
+
+        for mailing in mailings:
+            email_values = {'email_to': mailing.user_id.email}
+            mail_template.send_mail(mailing.id, email_values=email_values, force_send=True)
+
+    def get_statistics_email_values(self):
```

Comment:
@odony Yes, otherwise I can not call it in the mail template
![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/80179676-5e18e300-8601-11ea-9894-b80fe9f8c4b9.png)